sites.json

```diff
@@ -1521,9 +1521,9 @@
 
     {
         "name": "Discord",
-        "email": "support@discordapp.com",
-        "difficulty": "hard",
-        "notes": "Contact Discord’s customer support and request for your account to be closed.",
+        "url": "https://support.discordapp.com/hc/en-us/articles/212500837-How-do-I-permanently-delete-my-account-",
+        "difficulty": "easy",
+        "notes": "Follow Discord's guide.",
```

Comment:
This isn't especially useful; this should either be a copy of the relevant portion of their support page, `"After deleting or transferring ownership of any servers you own, you can go to \"User Settings -> My Account -> Edit -> Delete Account\""`, or should be removed. I'm actually in favor of removing the note and letting the support page speak for itself.

Reply:
That is a good point, I am in favor of removing the note too.